plugins/catalog-graphql/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@backstage/plugin-catalog-graphql",
+  "version": "0.1.1-alpha.17",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
```

Comment:
should this be cjs? i am not entirely sure about these things but the other backend plugins have that

Reply:
not sure - this is just copied from the `catalog-backend` @rugvip? does it matter?